app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
```

Comment:
Add comment on top of files with file overview so non ruby programmers can understand whats going on without having a headache.